packages/fields/src/actions/set-as-homepage.tsx

```diff
@@ -0,0 +1,247 @@
+/**
+ * WordPress dependencies
+ */
+import { select, useDispatch } from '@wordpress/data';
```

Comment:
We should avoid using the global "select" entirely because it can break easily, we're not sure that we're using the right data store by doing that. It's an API that is only meant for plugins and for very specific situations.